src/plugins/tableresize.js

```diff
@@ -163,6 +163,8 @@
 
 	function ColumnResizer(editor, pillar) {
 		let document;
+		let isResizing;
+		let move;
```

Comment:
These are another nasty example. Assigned somewhere in the middle of the function, hoisted to function scope, and used in some distant places.